docs/requirements-specification.md

```diff
@@ -21,4 +21,42 @@ The use cases fulfill the goals in the following way:
 | UC5 | G4, G8 |
 | UC6 | |
 
-The goals G10-G12 are currently not addressed by any use case.
\ No newline at end of file
+The goals G10-G12 are currently not addressed by any use case.
+
+### UC3: Get capability overview
+| UC3 | Get capability overview
+|---|---|
+| Actors | (Senior) Researcher |
+| Pre-conditions | The researcher is authenticated |
+| End conditions (success) | An overview of the capabilities of a research group is generated. |
+| End conditions (success) | An overview of the capabilities of a research group is generated. |
+| Steps | 1. If (Senior) Researcher presses "Research group overview" the system opens a new window with a table containing the following information about all the researchers belonging to the research group: researcher name, capability groups in which this researcher has capabilities, capabilities this researcher has, link to profile page. |
+| | 2. If (Senior) Researcher clicks on the page "Generate visualization" a simple visualization (piechart) of a proportion of table entries in columns capability groups, capabilities is shown. |
+| Extensions | 1.a. The (Senior) Researcher can filter table entries in each column. |
+| | 1.b. The (Senior) Researcher can sort table entries in each column. |
+
+
+
+| UC3 | Get capability overview
+|---|---|
+| Actors | Researcher |
+| Pre-conditions | The researcher is authenticated |
```

Comment:
This UC describes how a researcher *configures* their private portfolio visualization. There should also be a UC of an (unauthenticated) user that *accesses* this individual portfolio.